content/03.results.md

```diff
@@ -224,46 +235,47 @@ The mutual exclusivity of signatures 3 and MMR corroborates a previous report su
+We observed this trend in GSVA hallmark pathways (Bonferroni-corrected p < 0.05) (**Figure {@fig:Fig5}B**) and immune cell deconvolution, which showed that monocytes dominated the identified immune cells in these tumors (**Figure {@fig:Fig5}C**).
+We also observed significant upregulation of pro-inflammatory cytokines IFN$\alpha$ and IFN$\gamma$ in LGG and craniopharyngiomas compared to medulloblastoma and ependymoma tumors (Bonferroni-corrected p < 0.05), both of which showed significant down-regulation of these cytokines (**Figure {@fig:Fig5}B**).
+Together, these results supported previous proteogenomic findings of lower immune infiltration in aggressive medulloblastomas and ependymomas versus higher immune infiltration in _BRAF_-driven LGG and craniopharyngiomas [@doi:10.1016/j.cell.2020.10.044]. 
+
+
+Although CD8+ T-cell infiltration across all cancer groups was quite low (**Figure {@fig:Fig5}C**), we observed some signal in specific cancer molecular subtypes (medulloblastoma Groups 3 and 4) as well as outlier tumors (BRAF-driven LGG, BRAF-driven and wildtype ganglioglioma, H3 wildtype HGG; **Figure {@fig:S6}E**) 
+Surprisingly, the classically immunologically-cold HGG and DMG tumors [@doi:10.1186/s40478-018-0553-x; 10.1093/brain/awab155] contained higher overall fractions of immune cells, with monocytes, dendritic cells, and NK cells the most prevalent (**Figure {@fig:Fig5}C**).
+This finding suggests that quanTIseq might actually be capturing microglia within these immune cell fractions.
+While we did not detect notable prognostic effects of immune cell infiltration in HGG or DMG cancer groups, we did find that high levels of macrophage M1 and monocytes were associated with poorer overall survival (monocyte HR = 5.2e17, 95% CI = 1.2e5 - 2.3e30, p = 0.006) in medulloblastoma tumors (**Figure {@fig:Fig5}D**).
+We further reproduced previous findings (**Figure {@fig:Fig5}E**) that medulloblastomas typically have low expression of _CD274_ (PD-L1) [@doi:10.18632/oncotarget.24951].
+Consistent with a previous study [@doi:10.1080/2162402X.2018.1462430], we found higher expression of _CD274_ in the WNT subtype, which has the best prognosis of all medulloblastoma subgroups, compared to all other molecular subtypes (**Figure {@fig:Fig5}E**).
+However, we also found that higher expression of _CD274_ was significantly, although marginally, associated with improved overall prognosis (HR = 0.00074, 95% CI = 3.0e-6 - 0.18, p = 0.01) for medulloblastoma samples (**Figure {@fig:Fig5}D**)
+Finally, we asked whether any molecular subtypes might show an immunologically-hot phenotype, as roughly defined by a greater proportion of CD8+ to CD4+ T cells. **TODO: reference for this ratio?**
```

Comment:
Ha, ref is Jess Foster.... I can ask!

Reply:
Missed this again also - We can use this paper: [10.1136/jitc-2021-004012](http://dx.doi.org/10.1136/jitc-2021-004012)

Reply:
Also, this one: [10.4236/jct.2013.48164](http://dx.doi.org/10.4236/jct.2013.48164)